### DISCUSSION TITLE
Can a button be provided for paying users to manually obtain invoices?

### DISCUSSION THREAD

**Discussion by CrushiY** [Glench/ExtPay#208] [General]
Hi! I am one of the users of ExtensionPay and also a developer. Recently, my users asked me if they can obtain their own invoices instead of having to request them from me every month.
Regarding this requirement, I would like to know if it is feasible.

**Comment by burkybang** [comment]
Stripe can automatically email them

**Reply by CrushiY** [reply]
Thank you for your reply! I am a beginner, and I will go and do some research.
